packages/sdk/src/actor/physics/physicsBridge.ts

```diff
@@ -8,6 +8,7 @@ import { TransformLike } from "../transform";
 
 export interface PhysicsBridgeTransformInfo {
```

Comment:
I should have deleted whole interface when switched to binary stream for transforms.

Reply:
ah, so this is actually not needed.